Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -5542,6 +5956,261 @@ def create_gql_client(url_suffix="identity-protection/combined/graphql/v1"):
+
+    Returns:
+        dict[str, Any]: The raw response of the API.
+    """
+    query_params: dict[str, Any] = assign_params(service=service)
+    if policy_id:
+        query_params['policy-id'] = policy_id
+    if cloud_platform:
+        query_params['cloud-platform'] = cloud_platform
+    return http_request(method='GET', url_suffix='/settings/entities/policy/v1', params=query_params,
+                        status_code=[207])
+
+
+def cs_falcon_cspm_list_service_policy_settings_command(args: dict[str, Any]) -> CommandResults:
+    """Command to list service policy settings.
+
+    Args:
+        args (dict[str, Any]): The arguments of the command.
+
+    Returns:
+        CommandResults: The command results object.
+    """
+    policy_id = args.get('policy_id', '')
+    cloud_platform = args.get('cloud_platform', '')
+    service = args.get('service', '')
+    limit = arg_to_number(args.get('limit')) or 50
+    raw_response = cspm_list_service_policy_settings_request(policy_id=policy_id, cloud_platform=cloud_platform,
+                                                             service=service)
+    if resources := raw_response.get('resources', []):
+        # The API does not support pagination, therefore we have to do it manually
+        paginated_resources = resources[:limit]
+        human_readable = tableToMarkdown('CSPM Policy Settings:', paginated_resources,
+                                         headers=['policy_id', 'is_remediable',
+                                                  'remediation_summary', 'name', 'policy_type',
+                                                  'cloud_service_subtype', 'cloud_service', 'default_severity'],
+                                         headerTransform=string_to_table_header)
+        return CommandResults(readable_output=human_readable,
+                              outputs=paginated_resources,
+                              outputs_key_field='policy_id',
+                              outputs_prefix='CrowdStrike.CSPMPolicySetting',
+                              raw_response=raw_response)
+    return CommandResults(readable_output='No policy settings were found for the given arguments.')
+
+
+def cspm_update_policy_settings_request(account_id: str, enabled: bool, policy_id: int, regions: list[str],
+                                        severity: str, tag_excluded: bool | None) -> dict[str, Any]:
+    """Do an API call to update the policy settings.
+
+    Args:
+        account_id (str): The account ID.
+        enabled (bool): Whether to enable the policy or not.
+        policy_id (int): The policy ID.
+        regions (list[str]): The regions of the policy.
+        severity (str): The severity of the policy.
+        tag_excluded (bool | None): Whether to exclude tag or not.
+
+    Returns:
+        dict[str, Any]: The raw response of the API.
+    """
+    # https://assets.falcon.crowdstrike.com/support/api/swagger.html#/cspm-registration/UpdateCSPMPolicySettings
```

Comment:
intended?

Reply:
Yup, for future developers, to know from where we got the structure of the body, since the API doesn't specify this clearly

Reply:
Yup, since the API documentation does not specify clearly the body of the request.